docs/superpowers/specs/2026-04-17-ultraplan-phase-design.md

```
@@ -0,0 +1,160 @@
+- Return path via existing `/gsd-import --from <file>` (no new import logic)
+
+**Out of scope (future):**
+- Parallel next-phase planning during `/gsd-execute-phase`
+- Auto-detection of ultraplan's saved file path
+- Text mode / non-interactive fallback
+
+---
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)